tests/test-distribution/test-distribution-common/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -2222,17 +2223,25 @@ public void testLimitHandlers(String env) throws Exception
 
     @ParameterizedTest
     @ValueSource(strings = {"brotli", "gzip", "zstandard"})
-    public void testCompressionHandler(String encoding) throws Exception
+    public void testCompressionHandler(String compressionName) throws Exception
     {
         String jettyVersion = System.getProperty("jettyVersion");
         JettyHomeTester distribution = JettyHomeTester.Builder.newInstance()
             .jettyVersion(jettyVersion)
             .build();
 
+        String encoding = switch (compressionName)
+        {
+            case "brotli" -> "br";
+            case "gzip" -> "gzip";
+            case "zstandard" -> "zstd";
```

Comment:
```suggestion
            case "brotli" -> "br";
            case "gzip" -> "gzip";
            case "zstandard" -> "zstd";
            case "all" -> "br,compression-gzip,compression-zstd";
```

Reply:
@gregw I'm not sure what you exactly mean here.
You mean for "all" to have a `Content-Encoding` with just `br` as a valid value, and 2 other invalid values?
Or you meant all 3 valid values, so `br, gzip, zstd`?

Also, I just read RFC 9110, and it is way more complicated to support the right encoding than we can write in a distribution test, so I think it's best to write more complicated cases in non-distribution tests.

Reply:
I just think we need at least one distribution test that enables all the compression at once.  I'm OK if the test itself just sends a single algorithm in its accept header.    We just need to check the modules and xmls do not clash somehow.

I do mean all 3 valid values, I was just hacking in the "compression-", but a more elegant solution is possible